src/styles/notification.scss

```diff
@@ -1,7 +1,7 @@
 .notification {
   position: fixed;
   z-index: 1;
-  background: $primary-color;
+  background: blue;
```

Comment:
Maybe using variables is better idea, than just word like blue.

Reply:
Notification still need some work done 😕